tests/MongoDB.Driver.Tests/Specifications/client-side-encryption/prose-tests/ClientEncryptionProseTests.cs

```diff
@@ -760,6 +789,69 @@ public void ViewAreProhibitedTest([Values(false, true)] bool async)
             }
         }
 
+        // NOTE: this test is not presented in the prose tests
+        [SkippableTheory]
+        [ParameterAttributeData]
+        public void UnsupportedPlatformsTests(
+            [Values("local", "aws", "azure", "gcp")] string kmsProvider,
+            [Values(false, true)] bool async)
+        {
+            RequireServer.Check().Supports(Feature.ClientSideEncryption);
+
+            using (var clientEncrypted = ConfigureClientEncrypted())
+            using (var clientEncryption = ConfigureClientEncryption(clientEncrypted.Wrapped as MongoClient))
+            {
+                Guid? dataKeyId = null;
+                var dataKeyOptions = CreateDataKeyOptions(kmsProvider);
+                var exception = Record.Exception(() => dataKeyId = CreateDataKey(clientEncryption, kmsProvider, dataKeyOptions, async));
+                AsserResult(exception);
+            }
+
+            void AsserResult(Exception ex)
```

Comment:
Fix typo in name:

    void AssertResult(Exception ex)

Reply:
done